nextjs/sentry.server.config.ts

```diff
@@ -5,16 +5,21 @@
 import * as Sentry from "@sentry/nextjs";
 
 Sentry.init({
-  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+  dsn: process.env.SENTRY_DSN,
   environment: process.env.NODE_ENV,
```

Comment:
Can we use `process.env.NEXT_PUBLIC_ENVIRONMENT` (previously `SENTRY_ENV`) instead so that it aligns with Django's sentry env.